formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/ProtoBuf.kt

```diff
@@ -57,7 +57,7 @@ import kotlin.js.*
  * ### Usage example
  * ```
  * // Serialize to ProtoBuf hex string
- * val encoded = ProtoBuf.decodeToHexString(MyMessage(15)) // "080f1000182a"
+ * val encoded = ProtoBuf.decodeFromHexString(MyMessage(15)) // "080f1000182a"
```

Comment:
I'd still say `encodeToHexString` as `MyMessage` is not a hex string.

Reply:
:(

Reply:
fixed: https://github.com/Kotlin/kotlinx.serialization/commit/74653e164d18dcebe98d58c26781b7bf7f647e39